randomizer.py

```diff
@@ -998,6 +1036,26 @@ def potionCards(self):
 Traits = Plunder.traits
 LandscapeCards = Events | Landmarks | Projects | Ways | Traits
 
+# Define action cards
+ActionCards = (
+    Base.actionCards
+    | Intrigue.actionCards
+    | Seaside.actionCards
+    | Alchemy.actionCards
+    | Prosperity.actionCards
+    | Cornucopia.actionCards
+    | Hinterlands.actionCards
+    | DarkAges.actionCards
+    | Guilds.actionCards
+    | Adventures.actionCards
+    | Empires.actionCards
+    | Nocturne.actionCards
+    | Renaissance.actionCards
+    | Menagerie.actionCards
+    | Allies.actionCards
+    | Plunder.actionCards
+)
```

Comment:
We can future-proof this using the `AllSets` variable like so:

```python
listOfSetsOfActions = (cardSet.actionCards for cardSet in AllSets.values())
ActionCards = set().union(*listOfSetsOfActions)
```

Reply:
Or, y'know, `Actions = set().union(*listOfSetsOfActions)`, if my other review comment gets changed.

Reply:
Agreed.